### PR TITLE
CONSOLE-2430: i18n for cluster dashboard section of home nav

### PR DESCRIPTION
**Pull request by jcaianirh**
For: https://issues.redhat.com/browse/CONSOLE-2430

Cluster dashboard section only

### CODE REVIEW

File: frontend/packages/console-shared/src/components/dashboard/duration-hook.ts [hunk 1]
```diff
@@ -1,11 +1,34 @@
...
   TWENTY_FOUR_HR = '24 Hours',
 }
 
+export const TranslatedDuration: React.FC<TranslatedDurationProps> = ({ duration }) => {
+  let translatedString;
+  const { t } = useTranslation();
+
+  switch (duration) {
+    case Duration.ONE_HR:
+      translatedString = t('dashboard~1 Hour');
```
Comment (rebeccaalpert) [inline frontend/packages/console-shared/src/components/dashboard/duration-hook.ts:16]: UXD wants sentence case.
```suggestion
      translatedString = t('dashboard~1 hour');
```

File: frontend/packages/console-shared/src/components/dashboard/duration-hook.ts [hunk 2]
```diff
@@ -1,11 +1,34 @@
...
+      break;
+    case Duration.SIX_HR:
+      translatedString = t('dashboard~6 Hours');
```
Comment (rebeccaalpert) [inline frontend/packages/console-shared/src/components/dashboard/duration-hook.ts:19]:
```suggestion
      translatedString = t('dashboard~6 hours');
```

File: frontend/packages/console-shared/src/components/dashboard/duration-hook.ts [hunk 3]
```diff
@@ -1,11 +1,34 @@
...
+      break;
+    default:
+      translatedString = t('dashboard~24 Hours');
```
Comment (rebeccaalpert) [inline frontend/packages/console-shared/src/components/dashboard/duration-hook.ts:22]:
```suggestion
      translatedString = t('dashboard~24 hours');
```

File: frontend/public/components/dashboard/dashboards-page/cluster-dashboard/inventory-card.tsx [hunk 4]
```diff
@@ -120,10 +122,12 @@ export const InventoryCard = () => {
     replacementExtensions,
   ]);
 
+  const { t } = useTranslation();
+
   return (
     <DashboardCard data-test-id="inventory-card">
       <DashboardCardHeader>
-        <DashboardCardTitle>Cluster Inventory</DashboardCardTitle>
+        <DashboardCardTitle>{t('dashboard~Cluster Inventory')}</DashboardCardTitle>
```
Comment (rebeccaalpert) [inline frontend/public/components/dashboard/dashboards-page/cluster-dashboard/inventory-card.tsx:130]:
```suggestion
        <DashboardCardTitle>{t('dashboard~Cluster inventory')}</DashboardCardTitle>
```

File: frontend/public/components/dashboard/dashboards-page/cluster-dashboard/utilization-card.tsx [hunk 5]
```diff
@@ -429,7 +431,7 @@ export const UtilizationCard = () => {
   return (
     <DashboardCard data-test-id="utilization-card">
       <DashboardCardHeader>
-        <DashboardCardTitle>Cluster Utilization</DashboardCardTitle>
+        <DashboardCardTitle>{t('dashboard~Cluster Utilization')}</DashboardCardTitle>
```
Comment (rebeccaalpert) [inline frontend/public/components/dashboard/dashboards-page/cluster-dashboard/utilization-card.tsx:434]:
```suggestion
        <DashboardCardTitle>{t('dashboard~Cluster utilization')}</DashboardCardTitle>
```

**Review by rebeccaalpert — APPROVED** [review]
Looks good to me! Just a couple of sentence case things.

**Review by rebeccaalpert — APPROVED** [review]
Looks good to me! :) Thank you.

File: frontend/public/components/dashboard/dashboards-page/dashboards.tsx [hunk 6]
```diff
@@ -61,12 +63,12 @@ const DashboardsPage_: React.FC<DashboardsPageProps> = ({ match, kindsInFlight,
...
       {
         href: '',
-        name: 'Cluster',
+        name: t('dashboard~Cluster'),
         component: ClusterDashboard,
       },
       ...pluginPages,
     ],
-    [pluginPages],
+    [pluginPages, t],
```
Comment (spadgett) [inline frontend/public/components/dashboard/dashboards-page/dashboards.tsx:71]: Can you confirm that `t` does not change causing this to run again and again?

**Comment by spadgett** [comment]
/assign @yapei @ahardin-rh @sferich888 
for QE / docs / CEE review

This PR localizes the Home -> Overview page

**Comment by spadgett** [comment]
@jcaianirh Missing key:

> [SEVERE] https://console-openshift-console.apps.ci-op-m8g5wpqh-75d12.origin-ci-int-gce.dev.openshift.com/static/main-chunk-3f078185229fe71ba6ad.min.js 0:917791 "Missing i18n key \"Cluster utilization\" in namespace \"dashboard\" and language \"en.\""

@rebeccaalpert @dtaylor113 FYI, it looks like we're not logging the error message in the result:

> Error: Failed: omg js error: [object Object]

But I could find it in the browser.log for the Protractor tests. On the other hand, it's great CI is catching this!

@yapei Please hold on QE review until this is fixed. Thanks!

**Comment by dtaylor113** [comment]
> > Error: Failed: omg js error: [object Object]

This is being trapped [here](https://github.com/openshift/console/blob/fe3d8c48354d45dc39166fe0f77487058d253136/frontend/integration-tests/protractor.conf.ts#L273).  We process more of the `win.windowError` here in Cypress:  [here](https://github.com/openshift/console/blob/5464e1058c275f95a29688fbe2f5c192de0f78b7/frontend/packages/integration-tests-cypress/support/index.ts#L55)

File: frontend/public/components/dashboard/dashboards-page/dashboards.tsx (hunk 6 above)
Comment (jcaianirh) [reply frontend/public/components/dashboard/dashboards-page/dashboards.tsx:71]: @spadgett validated the HorizontalNav that uses allPages only renders once.

**Comment by ahardin-rh** [comment]
LGTM from the docs side

**Comment by jcaianirh** [comment]
>LGTM from the docs side

/label docs-approved

**Comment by yanpzhan** [comment]
/label qe-approved
Have tested against the pr, no issue is found for now.

**Review by spadgett — APPROVED** [review]
/lgtm

**Pull request merged**